src/benchmarks/gc/src/exec/GCPerfSim/GCPerfSim.cs

```diff
@@ -805,11 +804,6 @@ struct SizeSlot
     // an object size distribution for SOH/LOH derived from a real server scenario
     public static SizeSlot[] sohLohSizeSlots = new SizeSlot[]
     {
-//        new SizeSlot(        24,  16229766), We are creating some small wrapper objects of 32 and 40 bytes around the
-//        new SizeSlot(        32,  48790439), payload objects - somewhat compensate for this by distorting the distribution
-//        new SizeSlot(        40,  21496250), at the low end
-//        new SizeSlot(        48,  15020039),
-//
```

Comment:
I would not get rid of these - this is data from running real workloads - we should add the ability to create very small objects instead of getting rid of this.